mesheryctl/internal/cli/root/relationships/relationship.go

```diff
@@ -31,11 +31,18 @@ import (
 )
 
 var (
-	outFormatFlag string
-
-	availableSubcommands = []*cobra.Command{ViewRelationshipsCmd, GenerateRelationshipDocsCmd}
+	outFormatFlag        string
+	maxRowsPerPage       = 25
```

Comment:
the pagination logic and constants should be common to all commands

Reply:
I saw it was written in one of the command specifically [here](https://github.com/meshery/meshery/blob/7818e6fefae8deaa6a09961b6d0a984c0557ee6b/mesheryctl/internal/cli/root/environments/environment.go#L39) so I went ahead and declared for the whole relationship command.
It is 25 so i guess in a sense it is default but it should be manged from a single variable imo.